ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -16,12 +16,12 @@
     - ansible_architecture == "x86_64"
   tags: patch_update
 
-- name: Enable EPEL release for RHEL8 or RHEL6 or RHEL7
+- name: Enable EPEL release for RHEL9 or RHEL8 or RHEL6 or RHEL7
   yum: name=https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm
   failed_when: false
   when:
     - ansible_architecture != "s390x"
-    - (ansible_distribution_major_version == "8") or (ansible_distribution_major_version == "6") or (ansible_distribution_major_version == "7")
+    - (ansible_distribution_major_version | int >= 6)
```

Comment:
If this is needed for rhel6+ why do we even need a condition for it?

Reply:
No reason that I can think of 🤷🏻 
Looks like it initially had rhel 6 or rhel 8 and then 7 was added in under https://github.com/adoptium/infrastructure/pull/1480
Stripping out the condition altogether sounds good to me unless anyone can think of a good reason to leave it there.